OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -46,6 +48,13 @@ public enum ReadyTextStyleOptions { Solid, AlternatingColor, Blinking }
 		public readonly string TooltipContainer;
 		public readonly string TooltipTemplate = "PRODUCTION_TOOLTIP";
 
+		public readonly string ClockAnimation = ChromeMetrics.Get<string>("ProductionClockAnimation");
+		public readonly string ClockSequence = ChromeMetrics.Get<string>("ProductionClockSequence");
+		public readonly string ClockPalette = ChromeMetrics.Get<string>("ProductionClockPalette");
+		public readonly string NotBuildableAnimation = ChromeMetrics.Get<string>("ProductionNotBuildableAnimation");
+		public readonly string NotBuildableSequence = ChromeMetrics.Get<string>("ProductionNotBuildableSequence");
+		public readonly string NotBuildablePalette = ChromeMetrics.Get<string>("ProductionNotBuildablePalette");
```

Comment:
We only dropped that there for generic re-usable parts of the UI that is why I was asking. Also this will break every custom mods as you don't have upgrade rules to add the missing bits.